app/views/fileChecksProgress.scala.html

```diff
@@ -4,26 +4,35 @@
 
 @main("Checking records") {
     @defining(play.core.PlayVersion.current) { version =>
-    <div class="govuk-grid-row">
+    <div class="govuk-grid-row file-check-progress">
         <div class="govuk-grid-column-two-thirds">
 
             @progressIndicator(Messages("checkingRecords.progress"))
             @frontEndInputs(frontEndInfo)
-            <h1 class="govuk-heading-l">Checking records</h1>
-            <p class="govuk-body">We are now checking your records. It is safe to close the browser if you wish, otherwise please wait and you will be redirected automatically once complete.</p>
+            <h1 class="govuk-heading-l">Checking your records</h1>
             <input id="consignmentId" type="hidden" value="@consignmentId">
-            <div class="file-check-progress">
-                <label for="av-metadata-progress-bar" class="govuk-label">Anti-virus scanning</label>
-                <progress id="av-metadata-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.avMetadataProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="checksum-progress-bar" class="govuk-label">Validating data integrity</label>
-                <progress id="checksum-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.checksumProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="ffid-progress-bar" class="govuk-label">Identifying file formats</label>
-                <progress id="ffid-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.ffidMetadataProgressPercentage" max="100"></progress>
+            <p class="govuk-body">Please wait while your records are being checked. This may take a few minutes.</p>
+            <p class="govuk-body">The following checks are now being performed:</p>
+            <ul class="govuk-list govuk-list--bullet">
```

Comment:
Would it be okay to add the following class to the ul tag in order to have a bit more spacing under the bullet list? 
govuk-!-margin-bottom-7

It would then be the same spacing between list and disabled button and then later between list and the important box which also matches the spacing between the important box and the continue button.

Reply:
Scratch that. There was a paragraph missing. Could you add under the list the following:
<p class="govuk-body govuk-!-margin-bottom-8">For more information on these checks, please see our <a href="@routes.FaqController.faq()" class="govuk-link">FAQ</a> to this service.</p>

Reply:
I've added the class to the ul tag. I've left out the FAQ line though as we'll add it in later once the FAQ page is in.